### PR TITLE
[TreeView] Prepare support for PigmentCSS

### PR DESCRIPTION
**Pull request by flaviendelangle**
Part of #12278

### CODE REVIEW

**Review by brijeshb42 — APPROVED** [review]
+1 to the changes. cc: @siriwatknp since you've already done this for material-ui.

File: packages/x-tree-view-pro/src/RichTreeViewPro/RichTreeViewPro.tsx [hunk 1]
```diff
@@ -10,6 +9,7 @@ import {
   buildWarning,
   extractPluginParamsFromProps,
 } from '@mui/x-tree-view/internals';
+import { styled, createUseThemeProps } from '../internals/zero-styled';
```
Comment (siriwatknp) [inline packages/x-tree-view-pro/src/RichTreeViewPro/RichTreeViewPro.tsx:12]: 👍 This should work as long as it has `zero-styled`.

**Review by siriwatknp — APPROVED** [review]
✅ Looks good to me. Let me know when it is released, I'd love to try.

**Comment by flaviendelangle** [comment]
> ✅ Looks good to me. Let me know when it is released, I'd love to try.

I will be released this Thursday / Friday :+1:

**Pull request merged**